runtime/doc/develop.txt

```diff
@@ -251,6 +251,12 @@ Use existing common {action} names if possible:
     list    Get all things
     set     Set a thing (or group of things)
 
+Use existing common {thing} names if possible:
+    buf     Buffer
+    pos     Position
+    tab     Tabpage
```

Comment:
`tabpage` seems more common in API

Reply:
yeah, though I'm thinking we should change that eventually as we march towards 2.0.